### ISSUE TITLE
Running unit test after successful installation fails

### ISSUE THREAD

**Issue by MadejaJiri** [easybuilders/easybuild-framework#3448]
Disclaimer: I am a total newbie to this.

So after what seems to be a successful installation (sanity check seems ok), I tried [running unit tests](https://easybuild.readthedocs.io/en/latest/Installation.html#running-unit-tests) and it outputs a bunch of these:
WARNING: 'eb' not found in $PATH, failed to determine installation prefix!
Interspersed with various errors from time to time.

I don't want to bother with something that is certainly my fault, but after a fairly complicated installation and a prompt to "please open an issue to report it if not successful" and with my limited skill set, I would very much appreciate a hint.
Thank you.

**Comment by boegel** [comment]
@MadejaJiri The unit tests are known to have some issues w.r.t. the environment on which they are run.

In general, it's not really useful to run them after installing EasyBuild, because they won't tell you much about your EasyBuild installation. The tests are used while developing EasyBuild to make sure we don't accidentally break anything.

Since you closed this issue, I guess you managed to sort things out yourself?

**Comment by MadejaJiri** [comment]
Sorry, I closed the issue because I found the duplicit issue. [https://github.com/easybuilders/easybuild-framework/issues/3231](https://github.com/easybuilders/easybuild-framework/issues/3231)
Since the other issue was opened in March, I deemed it's not important for the functionality and decided to ignore it.
(Right now, I try to install Blender to try if it works, so far unsuccessfully. But I guess it's not connected to the issue).
